pytest.ini

```diff
@@ -1,7 +1,7 @@
 [pytest]
 # Fail if coverage falls below 95%
 # NOTE: If debug breakpoints aren't working, comment out the code coverage line below
-addopts = --cov samtranslator --cov-report term-missing --cov-fail-under 95
+# addopts = --cov samtranslator --cov-report term-missing --cov-fail-under 95
```

Comment:
Debug problems.. I've just removed this line to make debug work. (temporarily)

Reply:
yeh.. I'm thinking about that too.
So, maybe I will split this in 2 PR:
1 for setup.py
1 for tox (that will include the PEP8 compliant)

Reply:
Ya. Let's do that. I think it will make this easier to understand what changes are related. I think I can reason about it but still learning nuances of python/pip :)